### ISSUE TITLE
Orbit controls

### ISSUE THREAD

**Issue by giaco-leopardi** [Ultimaker/Cura#14090]
### Is your feature request related to a problem?

A way to change the orbit controls to be the same as in Fusion 360 or other 3D modeling software to make it easier to navigate in the workspace and don't mix up the controls when trying to pan or move the camera viewport

### Describe the solution you'd like

Adding the possibility to change the orbit controls to be the as as in other 3D modeling softwares in the preferences tab 

### Describe alternatives you've considered

adding customizable controls for panning and moving the camera viewport

### Affected users and/or printers

Users

**Comment by Sandsy78** [comment]
I agree that whilst every CAD softwre has its own weird navigation controls, I believe they should all always be customisable. The biggest problem in Cura is the Viewport navigation especially the orbit control. It seems to orbit sometimes around object, sometimes around build volume and sometimes around some random pivot off screen. Zoom control scrolling is far too jumpy and should have a "Shift" scroll for fine zoom. Im kinda getting over Cura and trying out other slicers because if this irritation.
